AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -58,8 +58,9 @@ struct ABMeasurementsView: View {
                 if session.isFollowed {
                     SessionLoadingView()
                 } else if session.isDormant {
-                    VStack {
+                    VStack(alignment: .leading, spacing: 8) {
                         measurementsTitle
+                        allCellMeasurementsTitle
```

Comment:
it's a complicated name, what would you say about `streamNames`?

Reply:
Cool, doing!